hack/package/deb/build.sh

```diff
@@ -63,3 +47,5 @@ apt-ftparchive release ./ > Release
 # if you want to use your gpg key, you should replace '439AE9EC' with your key
 gpg -abs --default-key 439AE9EC -o Release.gpg Release
 gpg --clearsign --default-key 439AE9EC -o InRelease Release
+
+echo "deb package was output to $PACKAGEDIR"
```

Comment:
How about:
echo "Build deb package successfully ! Please get the package in $PACKAGEDIR !"